config/locales/en.yml

```diff
@@ -771,19 +772,19 @@ en:
     client_logins:
       form:
         errors:
-          at_least_one: "Please fill out one of the two fields."
-          bad_confirmation_number: "Incorrect confirmation number. After 5 failed attempts, accounts are locked."
-          bad_last_four: "Incorrect last four. After 5 failed attempts, accounts are locked."
+          bad_input: "Incorrect client ID or last 4 of SSN/ITIN. After 5 failed attempts, accounts are locked."
       new:
         title: "To view your progress, we’ll send you a secure link."
+        one_form_of_contact: "Please enter 1 form of contact below:"
```

Comment:
I think we generally try to go quote-less in translation strings unless we need them because of the contents of the string. So like, one form of contact: would need quotes bc of the colon but a lot of the other ones wouldn't.

Reply:
It's true that that's our style. I have been ignoring this aspect of our style, hoping someone would run `i18n-tasks normalize` and fix all of them at once. :)